src/main/java/org/team1540/robot2023/utils/ButtonPanel.java

```diff
@@ -0,0 +1,109 @@
+package org.team1540.robot2023.utils;
+
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
+public class ButtonPanel extends GenericHID {
+
+    public enum PanelButton {
+        TOP_LEFT(1),
+        TOP_CENTER(2),
+        TOP_RIGHT(3),
+        MIDDLE_LEFT(4),
+        MIDDLE_CENTER(5),
+        MIDDLE_RIGHT(6),
+        BOTTOM_LEFT(7),
+        BOTTOM_CENTER(8),
+        BOTTOM_RIGHT(9);
```

Comment:
Is the button panel going to have lighting controls? If so you might want to add them here.